requirements/index.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
```

Comment:
Extra spaces appear in the green diff version reviewed, recommendation is to remove extra spaces throughout the document, where applicable.

Reply:
Done.